src/dp/distributions.rs

```diff
@@ -570,7 +570,7 @@ mod tests {
             .unwrap();
 
         // collect that number of samples
-        let mut rng = SeedStreamTurboShake128::from_seed([0u8; 16]);
+        let mut rng = SeedStreamTurboShake128::from_seed([1u8; 32]);
```

Comment:
@divergentdave this test seems to be pretty sensitive to the seed's value. Originally I had the seed set to `[0u8; 32]`, but the test fails with that value. In fact the probability was significantly lower than `alpha`.

This code is only called by `empirical_test_gauss`. What's the value of this test?

Reply:
It looks like you just got unlucky with the seed, so changing it is fine. I re-ran the test 1000 times with random seeds, and got a 96% success rate. We specify a significance level (`alpha`) of 0.05, so this shows our test is well-calibrated. This function performs a chi-squared statistical test which compares a histogram drawn from our discrete Gaussian sampler, and an expected histogram produced by evaluating the discrete Gaussian's cumulative distribution function on the same buckets. The test tells us that the sampler produces a distribution that is approximately correct. (We don't want to do more trials/samples to get a more powerful statistical test because this unit test will typically be running in debug mode, and its runtime is already pushing it)